include/mbedtls/sha256.h

```diff
@@ -49,81 +48,88 @@ extern "C" {
 #endif
 
 /**
- * \brief          SHA-256 context structure
+ * \brief          The SHA-256 context structure.
```

Comment:
Suggestion: We might want to add here something along the lines of `The structure is used both for SHA-256 and for SHA-224 checksum calculations. The choice between these two is made in the call to mbedtls_md_starts_ret().`

Reply:
Added as detailed desc.

Reply:
Sorry for the typo in the suggestion.